.circleci/config.yml

```diff
@@ -152,27 +152,23 @@ commands:
       k8s_image_name:
         type: string
         default: "deriv-app"
+      k8s_namespace:
+        type: string
+        default: "deriv-app-staging"
     steps:
       - k8s/install-kubectl
       - run:
-          name: Deploying to k8s cluster for service deriv-app
+          name: Deploying to k8s cluster for service << parameters.k8s_namespace >>
```

Comment:
Think this should say _svc_name_ to _k8s_namespace_

Reply:
`k8s_svc_name` parameter used by old deployment and will remove after successfully switch to `blue/green`

Reply:
The name of step edited.